### PR TITLE
feat: Endevor Bridge for Git integration

### PR DESCRIPTION
**Pull request by slavek-kucera**

### CODE REVIEW

File: clients/vscode-hlasmplugin/schema/proc_grps.schema.json [hunk 1]
```diff
@@ -36,6 +36,11 @@
                                             "items": {
                                                 "type": "string"
                                             }
+                                        },
+                                        "prefer_alternate_root": {
+                                            "type": "boolean",
+                                            "description": "Indicates that relative paths should be evaluated with respect to an alternative root folder (e.g. provided by the Endevor Bridge for Git).",
```
Comment (zeibura) [inline clients/vscode-hlasmplugin/schema/proc_grps.schema.json:42]: Specify "true" to resolve relative paths with respect to a different root folder than the workspace root (e.g. a folder provided by Endevor Bridge for Git)

File: clients/vscode-hlasmplugin/README.md [hunk 2]
```diff
@@ -53,6 +53,8 @@ To do this, set up two configuration files — `proc_grps.json` and `pgm_conf.js
 
 Follow the section [External Macro Libraries and COPY Members](#External-Macro-Libraries-and-COPY-Members) below for more detailed instructions on configuring the environment.
 
+The `pgm_conf.json` file may be provided implicitly by another product that supports integration with the HLASM Language support (e.g. Endevor Bridge for Git integration).
```
Comment (zeibura) [inline clients/vscode-hlasmplugin/README.md:56]: The `pgm_conf.json` file can be provided implicitly by another product that supports integration with HLASM Language Support (e.g. Endevor Bridge for Git).

**Review by zeibura — CHANGES_REQUESTED** [review]

File: parser_library/src/workspaces/workspace.cpp [hunk 3]
```diff
@@ -24,7 +24,7 @@
 #include <unordered_map>
 #include <unordered_set>
 
-#include "file_manager_vfm.h"
+#include "file_manager.h"
```
[the remark this answers is not in the document]
Comment (slavek-kucera) [reply parser_library/src/workspaces/workspace.cpp:27]: file_manager_vfm is needed

[the remark this answers is not in the document]
Comment (jirimosinger) [reply parser_library/src/workspaces/workspace.cpp:27]: I've meant also the rest of the includes but github didn't allow me to select the lines which were not edited...

File: parser_library/src/workspaces/workspace_configuration.cpp [hunk 4]
```diff
@@ -0,0 +1,709 @@
...
+    return nullptr;
+}
+
+bool workspace_configuration::try_loading_alternative_configuration(
+    const utils::resource::resource_location& file_location)
+{
+    const auto b4g_conf = utils::resource::resource_location::replace_filename(file_location, B4G_CONF_FILE);
+
+    if (m_b4g_config_cache.contains(b4g_conf))
+        return false;
```
[the remark this answers is not in the document]
Comment (slavek-kucera) [reply parser_library/src/workspaces/workspace_configuration.cpp:685]: the function simply says if something *changed*.

[the remark this answers is not in the document]
Comment (jirimosinger) [reply parser_library/src/workspaces/workspace_configuration.cpp:685]: Alright

File: parser_library/src/workspaces/workspace.cpp (hunk 3 above)
[the remark this answers is not in the document]
Comment (slavek-kucera) [reply parser_library/src/workspaces/workspace.cpp:27]: been looking into `workspace.h`... I'll recheck.

**Pull request merged**